clients/admin-ui/src/features/user-management/user-management.slice.ts

```diff
@@ -167,13 +152,13 @@ export const userApi = createApi({
       invalidatesTags: ["User"],
     }),
     updateUserPermissions: build.mutation<
-      UserPermissionsUpdate,
-      Partial<UserPermissionsUpdate> & Pick<UserPermissionsUpdate, "id">
+      UserPermissions,
+      UserPermissionsEditParams
     >({
-      query: ({ id, scopes }) => ({
-        url: `user/${id}/permission`,
+      query: ({ user_id, scopes }) => ({
+        url: `user/${user_id}/permission`,
         method: "PUT",
-        body: { id, scopes },
+        body: { id: user_id, scopes },
```

Comment:
🤪 I renamed this parameter to `user_id` because that's what's being passed in. However, the API intends for the body to include the `id` of the UserPermissions record.  Technically this works fine -- it's been working for a while --  because there's nothing wrong with the two records having the same id string. However, I think this was unintentional.